report/schemas.api.md

```diff
@@ -462,6 +462,7 @@ export type Item = {
     available: number;
     isOnSale: boolean;
     creator: string;
+    beneficiary: string | null;

```

Comment:
@cazala @juanmahidalgo for retro-compatibility reasons prefer using `beneficiary?: string;` instead of `beneficiary: string | null;` the second implies that the key `beneficiary`